--- workflows/haskell-ci.yml	2022-01-31 17:55:16.000000000 +0100
+++ workflows/haskell-ci.yml-patched	2022-01-31 17:59:20.000000000 +0100
@@ -14,8 +14,14 @@
 #
 name: Haskell-CI
 on:
-  - push
-  - pull_request
+  push:
+    branches:
+      - master
+      - experimental
+  pull_request:
+    branches:
+      - master
+      - experimental
 jobs:
   linux:
     name: Haskell-CI - Linux - ${{ matrix.compiler }}
